### DISCUSSION TITLE
unknown modification error using the mouse library from SwathAtlas

### DISCUSSION THREAD

**Discussion by pejota66** [vdemichev/DiaNN#721] [General]
Hello,
we are using the Mouse library from SwathAtlas. When we want to use DIA-NN with this public library, it shows the following error
C:\diann\src\diann.cpp: 2749: unknown modification: Oxidation (P)

I have tried to use “—mod” on the additional comments tab but unsuccessfully. Could you please help me how to add "--mod Oxidation (P)" into the additional options?

Many thanks.

**Comment by vdemichev** [comment]
-- mod Oxidation (P),15.994915
Or alternatively just change Library generation to IDs, RT and IM

**Comment by pejota66** [comment]
Hello Vadim,

many thanks for your help.
by adding the "-- mod Oxidation (P),15.994915", the error still persists. 
On the other hand, change Library generation to IDs, RT and IM worked well.
